### PR TITLE
Add completion of simulator names on setup_pocs command line.

### PR DESCRIPTION
**Pull request by jamessynge**
Add help (docstring) to commands in PocsShell where missing.
Cleanup comments.

### CODE REVIEW

File: bin/pocs_shell [hunk 1]
```diff
@@ -75,14 +75,14 @@ class PocsShell(Cmd):
...
-        """Enter the drift alignment shell """
+        """Enter the drift alignment shell."""
         self.do_reset_pocs()
         print_info('*' * 80)
         i = DriftShell()
         i.cmdloop()
 
     def do_start_messaging(self, *arg):
-        """ Starts the messaging system for the POCS ecosystem
+        """Starts the messaging system for the POCS ecosystem
```
Comment (wtgee) [inline bin/pocs_shell]: Period here too.

I wish there were a way in reviews to make very simple edits like this...

File: bin/pocs_shell [hunk 2]
```diff
@@ -156,11 +156,20 @@ simulate is a space-separated list of hardware to simulate.
 Hardware names: {}   (or all for all hardware)'''.format(
             ','.join(hardware.get_all_names())))
 
+    def complete_setup_pocs(self, text, line, begidx, endidx):
```
Comment (wtgee) [inline bin/pocs_shell]: Can we move this down to the other Utility methods section? The organization scheme is entirely my own across a lot of our files but I think makes sense. Let me know if disagree.

Comment (wtgee) [reply bin/pocs_shell]: I only see the `text` param being used. Are the others still required?

File: bin/pocs_shell [hunk 3]
```diff
@@ -156,11 +156,20 @@ simulate is a space-separated list of hardware to simulate.
...
+        Does NOT park the mount, nor execute power_down.
+        """
         self.pocs = None
 
     def do_run_pocs(self, *arg):
-        """ Make POCS `run` in a separate process
+        """Make POCS `run` in a separate process
 
         Note:
             Run with `--simulator=all` for simulator
```
Comment (wtgee) [inline bin/pocs_shell]: Should update this comment to reflect how `simulator` is used.

File: bin/pocs_shell [hunk 4]
```diff
@@ -233,32 +242,37 @@ Hardware names: {}   (or all for all hardware)'''.format(
...
         try:
             self.pocs.observatory.mount.park()
             self.pocs.say("Mount parked")
         except Exception as e:
-            print_warning('Problem unparking: {}'.format(e))
+            print_warning('Problem parking: {}'.format(e))
 
     def do_go_home(self, *arg):
-        """ Move the mount to home """
+        """Move the mount to home"""
```
Comment (wtgee) [inline bin/pocs_shell]: Tricky periods. :)

File: bin/pocs_shell (hunk 2 above)
Comment (jamessynge) [reply bin/pocs_shell]: They are all passed in by Cmd when it asks for a set of completions. I assume that Python would complain if we didn't declare them.

Comment (jamessynge) [reply bin/pocs_shell]: Regarding moving the completion_* (and help_*) methods, I'd like to keep those methods grouped together by command name. Perhaps we should add a delineating comment (i.e. ####...) between commands?

File: bin/pocs_shell (hunk 3 above)
Comment (jamessynge) [reply bin/pocs_shell]: I don't know what to update it with. The comment looks wrong, but I've not executed this command enough to know what to update this with... except the missing period at the end! ;-)

File: bin/pocs_shell (hunk 1 above)
Comment (jamessynge) [reply bin/pocs_shell]: Done.

File: bin/pocs_shell (hunk 4 above)
Comment (jamessynge) [reply bin/pocs_shell]: Done.

File: bin/pocs_shell (hunk 3 above)
Comment (wtgee) [reply bin/pocs_shell]: Back in https://github.com/panoptes/POCS/commit/3cf2831b0dda9ecf3b8d70ab64069789d06a4c94#diff-f081d3c1efc3be3fbd0aa696e79e797a commit you had switched it do a space-separated list and without the `--simulator=`. The `Note:` here refers to commands to pass to `setup_pocs`, which is probably confusing.

**Pull request merged**